raspbian/raspian-stretch-lite-setup.md

```diff
@@ -0,0 +1,131 @@
+
+
+First of all go to [Download Raspbian from official raspberry pi website](https://www.raspberrypi.org/downloads/raspbian/). We will use the NON desktop lite version because We only need access to command line and not the graphical user interface.
+
+The easier method to install on the SD card is using Etcher. Go [here](https://etcher.io/) to download Etcher and install it. 
+
+* Connect the SD card.
+
+* Open Etcher and choose the image file (.zip) that you just downloaded (The filename is something like this: 201x-xx-xx-raspbian-stretch-lite.zip).
+
+* Select the SD card you want to write to. And click “Flash!”.
+
+**Enable WiFi**
+
+With the SD card connected. In the root of the SD card create a new file named: `wpa_supplicant.conf`
+
+In the file, Add:
+
+```
+country=US
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev update_config=1 
+network={ 
+    scan_ssid=1 
+    ssid="*NAME*" 
+    psk="*PASSWORD*" 
+    }
+```
+
+* country should be your country code. Change if you are outside of US
+* ssid is the name of the Wi-Fi network.
+* psk is the password of your Wi-Fi Network
+
+**Enable SSH**
+
+Create an empty file in the root of the SD card named `ssh` (withOUT dot or any extension). 
+
+`touch ssh`
+
+This will enable ssh on your Raspberry Pi.
+
+Put the SD card into Raspberry Pi and plug in the Power cable and wait a little bit so the raspberry boot and connect automatically to Wi-Fi.
+
+
+### Connect via SSH:
+
+Find the IP Address of your Raspberry Pi.
+
+`ssh pi@<IP>`
+
+`pwd` will show `/home/pi`
+
+**Set Locale**
+
+* Edit /etc/locale.gen and uncomment the line with en_US.UTF-8
+
+* `sudo locale-gen`
+
+> locale-gen reads `/etc/locale.gen` file to know what locales to generate. 
+
+* add `LANGUAGE=en_US.UTF-8` to `/etc/default/locale`
+
+* add `LC_ALL=en_US.UTF-8` to `/etc/default/locale`
+
+After restart you will see the new locale. Run `locale` to view the set locales. 
+
+### Setting up FogLAMP
+
+From your dev machine, `scp raspbian/deploy-core.sh pi@<IP>:/home/pi/`
+
+SSH to Raspbian machine and:
+
+Run `./deploy-core.sh` inside rasperry pi.
+
+
+```
+git clone https://github.com/foglamp/FogLAMP.git
+
+cd FogLAMP
+
+export FOGLAMP_ROOT=`pwd`
+
+make
+
+./scripts/foglamp start
+```
+
+### Setting up FogLAMP gui
+
+**On dev machine:**
+
+[Set up dev env](https://github.com/foglamp/foglamp-gui#prerequisites)
+
+`git clone https://github.com/foglamp/foglamp-gui.git` 
+
+> default will be on develop branch
+
+`cd foglamp-gui/`
+
+run `./build.sh`
+
+**Copy compressed build file and deploy script to Raspbian**
+
+`scp foglamp-gui-1.2.0.tar.gz pi@1<IP>:/home/pi/`
+
+> scp deploy.sh pi@192.168.0.11:/home/pi/
+
+> pi@192.168.0.11's password:
+
+> deploy.sh                   100% 4633   133.5KB/s   00:00
+
+`scp deploy.sh pi@<IP>:/home/pi/`
+
+> $ scp foglamp-gui-1.2.0.tar.gz pi@192.168.0.11:/home/pi/
+
+> pi@192.168.0.11's password:
+
+> foglamp-gui-1.2.0.tar.gz    100%  460KB   1.7MB/s   00:00
```

Comment:
I need to fix these example output ; it got swapped with `foglamp-gui-1.2.0.tar.gz` copy